Directory.Packages.props

```diff
@@ -126,7 +126,7 @@
     <!-- unit test dependencies -->
     <PackageVersion Include="Microsoft.Extensions.Diagnostics.Testing" Version="8.0.0" />
     <!-- Pinned version for Component Governance - Remove when root dependencies are updated -->
-    <PackageVersion Include="Newtonsoft.Json" Version="13.0.3" /> <!-- Introduced by Microsoft.Azure.Cosmos, https://github.com/Azure/azure-cosmos-dotnet-v3/issues/4302 -->
```

Comment:
Revert this. Or at least put the comment above what it applies to. As this is it doesn't make sense.

Reply:
I didn’t add it but the file got auto formatted